static/js/virtual_classroom.js

```
@@ -0,0 +1,1508 @@
+            document.getElementById(`${e.target.id}Value`).textContent = e.target.value;
+            state.settings = settings;
+            saveSettings(settings);
+        });
+    });
+
+    // Range input handlers
+    document.querySelectorAll('input[type="range"]').forEach(input => {
+        input.addEventListener('input', (e) => {
+            const settings = loadSettings();
+            const settingKey = e.target.id.replace(/([A-Z])/g, '_$1').toLowerCase();
+            settings[settingKey] = parseInt(e.target.value);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)